src/sdk/ConversationTranslator/Conversation.ts

```diff
@@ -175,56 +241,36 @@ export class ConversationImpl extends Conversation implements IDisposable {
 
         // save the config properties
         const configImpl = speechConfig as SpeechTranslationConfigImpl;
+        Contracts.throwIfNull(configImpl, "speechConfig");
         this.privProperties = configImpl.properties.clone();
-
+        this.privIsConnected = false;
+        this.privParticipants = new InternalParticipants();
+        this.privIsReady = false;
     }
 
-    /***
+    /**
      * Create a new conversation as Host
+     * @param cb
+     * @param err
      */
     public createConversationAsync(cb?: () => void, err?: (e: string) => void): void {
```

Comment:
Should be a public static method for consistency with the C/C++/C# code

Reply:
The abstract class is static so you can call it like this:
conversation = SpeechSDK.Conversation.createConversationAsync(config, cb etc).